telegram/ext/basepersistence.py

```diff
@@ -446,6 +446,8 @@ def get_callback_data(self) -> Optional[CDCData]:
 
         .. versionadded:: 13.6
 
+        .. versionchanged:: 14.0
```

Comment:
mh, this now just states that __something__ changed, but not what …

Reply:
Ohh, sorry i didn't knew much about sphinx doc directives..